lib/tasks/utilities.rake

```diff
@@ -33,24 +33,8 @@ namespace :utilities do
     BREAKPOINTS = [nil, "sm", "md", "lg", "xl"].freeze
     # rubocop:enable Lint/ConstantDefinitionInBlock
 
-    utility_data =
-      JSON.parse(
-        File.read(
-          File.expand_path(File.join(*%w[.. .. node_modules @primer css dist stats utilities.json]), __dir__)
```

Comment:
I'm not 100% certain, but we might not need `primer/css` in the root `package.json` after this change? We do still need it in the `demo/` and `docs/` apps though